sdks/go/pkg/beam/core/typex/special.go

```diff
@@ -36,6 +36,7 @@ var (
 
 	EventTimeType = reflect.TypeOf((*EventTime)(nil)).Elem()
 	WindowType    = reflect.TypeOf((*Window)(nil)).Elem()
+	TimersType    = reflect.TypeOf((*Timers)(nil)).Elem()
```

Comment:
This seems like the only place that `Timers` is used (at least in this PR). Why can't this be TimersMap?

Reply:
TimerMap is just a placeholder for timer details but Timers is the actual type used in standard timer coder.